plugin/vcs/gitlab/gitlab.go

```diff
@@ -251,37 +251,47 @@ func (p *Provider) ExchangeOAuthToken(ctx context.Context, instanceURL string, o
 	return oauthResp.toVCSOAuthToken(), nil
 }
 
-// FetchAllRepositoryList fetches all repositories where the authenticated user has a maintainer role.
-// The maintainer role is required to create webhook in the project.
+// FetchAllRepositoryList fetches all repositories where the authenticated user
+// has a maintainer role, which is required to create webhook in the project.
+//
+// Docs: https://docs.gitlab.com/ee/api/projects.html#list-all-projects
 func (p *Provider) FetchAllRepositoryList(ctx context.Context, oauthCtx common.OauthContext, instanceURL string) ([]*vcs.Repository, error) {
```

Comment:
No logic change to this method, just to match the pattern with https://github.com/bytebase/bytebase/pull/1423

Reply:
SG